compiler/luci-interpreter/src/Interpreter.cpp

```diff
@@ -125,6 +125,21 @@ void Interpreter::readOutputTensor(const luci::CircleOutput *output_node, void *
     tensor->readData(data, data_size);
 }
 
+size_t Interpreter::getOutputTensorSize(const luci::CircleOutput *output_node)
+{
+  Tensor *tensor = _runtime_module->getOutputTensors()[output_node->index()];
+  if (tensor == nullptr)
+  {
+    const std::string &name = output_node->name();
+    throw std::runtime_error("Cannot find tensor size for output node named \"" + name + "\".");
+  }
+
+  size_t tensor_size = luci::size(output_node->dtype());
```

Comment:
It would be better to use tensor API.

```suggestion
  size_t tensor_size = luci_interpreter::size(tensor->element_type());
```

Reply:
@jinevening,
Thanks for the suggestion!

Reply:
@jinevening,

The `luci_interpreter::size()` doesn't seem to exist.
Should I make this one..? or do I miss something..?

```console
/git/ONE/compiler/luci-interpreter/src/Interpreter.cpp:137:42: error: ‘size’ is not a member of ‘luci_interpreter’
  137 |   size_t tensor_size = luci_interpreter::size(tensor->element_type());
```

Reply:
maybe it's `luci::size(dtype)`

Reply:
Ah, sorry for confusion. I've seen different interface in luci_interpreter in onert-micro.

`luci-interpreter::getDataTypeSize()` may be useful. :)